Makefile

```diff
@@ -1,19 +1,24 @@
 BRANCH = "master"
 BUILD_TIME = "$(shell date -u +\"%Y-%m-%dT%H:%M:%SZ\")"
-VERSION = $(shell cat ./VERSION)
 REPONAME = "neo-go"
 NETMODE ?= "privnet"
 
-build:
-	@echo "=> Building darwin binary"
-	@go build -i -ldflags "-X github.com/CityOfZion/neo-go/config.Version=${VERSION}-dev -X github.com/CityOfZion/neo-go/config.BuildTime=${BUILD_TIME}" -o ./bin/neo-go ./cli/main.go
+REPO ?= `go list -m`
+VERSION ?= `git describe --tags 2>/dev/null || git rev-parse --short HEAD`
```

Comment:
That looks like a separate issue, not related to docker itself, thus at least deserving its own commit. Also, to make it really useful we would need some `vX.Y.Z-dev` tags.

Reply:
- Before `VERSION = $(shell cat ./VERSION)`
- Now `VERSION ?= $(git describe --tags 2>/dev/null || git rev-parse --short HEAD)`

there's no `./VERSION` file in repo and build fails..  needs pass version into docker-build